package.json

```diff
@@ -19,7 +20,8 @@
     "setup": "npm install && npm run validate -s"
   },
   "files": [
-    "dist"
+    "dist",
+    "typings"
```

Comment:
Could you rebase your branch on master? There will probably be a small conflict here.